src/natcap/invest/ui/sdr.py

```diff
@@ -123,6 +123,14 @@ def __init__(self):
             label='Max SDR Value',
             validator=self.validator)
         self.add_input(self.sdr_max)
+        self.l_max = inputs.Text(
+            args_key='l_max',
+            helptext=(
+                'L will not exceed this value. Ranges of 122-333m are found '
```

Comment:
Does this need to be unitless, not meters?

Reply:
changed to unitless in 624d4139